build.gradle

```diff
@@ -0,0 +1,119 @@
+plugins {
+    id 'java'
+    id 'eu.xenit.amp' version '1.0.1'
+    id 'eu.xenit.alfresco' version '1.0.1'
+    id "maven-publish"
+    id "signing"
+    id "idea"
+
+}
+
+description = 'Arender amp for Alfresco'
+group = 'eu.xenit.transformers'
+
+
+def baseVersion = '1.0.0'
+def branchName = System.env.TRAVIS_BRANCH
+def isRelease = branchName != null && branchName.startsWith("release")
+if (isRelease)
+    version = "$baseVersion"
+else
+    version = "$baseVersion-SNAPSHOT"
+
+
+repositories {
+    mavenCentral()
+    alfrescoPublic()
+}
+
+
+ext.alfrescoVersion = "5.1.e"
+
+
+
+test {
+    useJUnitPlatform()
+}
+
+dependencies {
+    compile group: 'org.alfresco', name: 'alfresco-platform-distribution', version: '5.1.e', ext: 'pom'
```

Comment:
I think this dependency is not necessary.
All necessary dependencies are already added by depending on `org.alfresco:alfresco-repository`